pyiron_nodes/atomistic/engine/vasp.py

```diff
@@ -0,0 +1,263 @@
+    print(f"Running job in directory: {workdir}")
+    output = ShellOutput()
+    output.stdout = proc.stdout
+    output.stderr = proc.stderr
+    print(proc.stdout, proc.stderr)
+    output.return_code = proc.returncode
+    return output
+
+
+@Workflow.wrap.as_function_node("output_dict")
+def parse_VaspOutput(workdir: str) -> Dict:
+    print(f"workdir of parse: {workdir}")
+    return pvo(workdir)
+
+
+@Workflow.wrap.as_function_node("convergence")
+def check_convergence(
+    workdir: str,
+    filename_vasprun: str = "vasprun.xml",
+    filename_vasplog: str = "vasp.log",
+    backup_vasplog: str = "error.out",
+) -> bool:
+    converged = False
+    line_converged = (
+        "reached required accuracy - stopping structural energy minimisation"
+    )
+
+    try:
+        vr = Vasprun(filename=os.path.join(workdir, filename_vasprun))
+        converged = vr.converged
+    except:
+        try:
+            converged = is_line_in_file(
+                filepath=os.path.join(workdir, filename_vasplog),
+                line=line_converged,
+                exact_match=False,
+            )
+        except:
+            try:
+                converged = is_line_in_file(
+                    filepath=os.path.join(workdir, backup_vasplog),
+                    line=line_converged,
+                    exact_match=False,
+                )
+            except:
+                pass
+
+    return converged
+
+
+@Workflow.wrap.as_macro_node("vasp_output", "convergence_status")
+def vasp_job(
+    self,
+    workdir: str,
+    vasp_input,
+    command: str = "module load vasp; module load intel/19.1.0 impi/2019.6; unset I_MPI_HYDRA_BOOTSTRAP; unset I_MPI_PMI_LIBRARY; mpiexec -n 1 vasp_std",
+):
+    self.working_dir = create_WorkingDirectory(workdir=workdir)
+    self.vaspwriter = write_VaspInputSet(workdir=workdir, vasp_input=vasp_input)
+    self.job = run_job(command=command, workdir=workdir)
+    self.vasp_output = parse_VaspOutput(workdir=workdir)
+    self.convergence_status = check_convergence(workdir=workdir)
+
+    (
+        self.working_dir
+        >> self.vaspwriter
+        >> self.job
+        >> self.vasp_output
+        >> self.convergence_status
+    )
+    self.starting_nodes = [self.working_dir]
+    return self.vasp_output, self.convergence_status
+
+
+def stack_element_string(structure) -> Tuple[List[str], List[int]]:
+    # site_element_list = [atom.symbol for atom in atoms]
+    site_element_list = [site.species_string for site in structure]
+    past_element = site_element_list[0]
+    element_list = [past_element]
+    element_count = []
+    count = 0
+
+    for element in site_element_list:
+        if element == past_element:
+            count += 1
+        else:
+            element_count.append(count)
+            element_list.append(element)
+            count = 1
+            past_element = element
+
+    element_count.append(count)
+    return element_list, element_count
+
+
+def get_default_POTCAR_paths(
```

Comment:
I haven't followed the logic to load the potentials in detail yet, but it'd be good to think already how to matches with the current way pyiron resources are set up to make the transition easy.  It doesn't need to be a one on one match.  Would be nice to use the [resolvers](https://github.com/pyiron/pyiron_snippets/blob/main/pyiron_snippets/resources.py) from snippets.